keel-sql/src/test/resources/application.properties

```diff
@@ -1,3 +1,4 @@
+scheduling.enabled=false
```

Comment:
Explicitly disable scheduling in `keel-sql` tests

Reply:
I remember trying to debug this myself several months ago and seeing that the annotations didn't take effect, but I also have this vague memory that even the config property wasn't enough, because kork was enabling scheduling programmatically and it was hard-coded. Are you positive this does the trick? If not, I can go dig that code in kork again.